cmd/serve.go

```diff
@@ -175,7 +175,9 @@ func (s *ServerService) SetupConsumers(ctx context.Context, o SetupConsumersOpti
 		return fmt.Errorf("creating Payment Completed Kafka Consumer: %w", err)
 	}
 
-	paymentReadyToPayConsumer, err := events.NewKafkaConsumer(
+	// Both Stellar and Cicle have a readyToPayConsumer that reads from the `PaymentsReadyToPayTopic`.
+	// We separate them to avoid the noisy neighbor problem.
+	stellarPaymentReadyToPayConsumer, err := events.NewKafkaConsumer(
```

Comment:
This comment seems outdated. Now, Circle reads from a separate topic right?

Reply:
addressed in [7e3a652](https://github.com/stellar/stellar-disbursement-platform-backend/pull/366/commits/7e3a6529726351538e3539db5f5e302d3f782e86)